ec/ecresource/organizationresource/schema.go

```diff
@@ -0,0 +1,250 @@
+			"id": schema.StringAttribute{
+				MarkdownDescription: "Organization ID",
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"members": organizationMembersSchema(),
+		},
+	}
+}
+
+func organizationMembersSchema() schema.MapNestedAttribute {
+	return schema.MapNestedAttribute{
+		MarkdownDescription: "Manages the members of an Elastic Cloud organization. The key of each entry should be the email of the member.",
+		Optional:            true,
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: map[string]schema.Attribute{
+				"email": schema.StringAttribute{
+					MarkdownDescription: "Email address of the user.",
+					Computed:            true,
+					PlanModifiers: []planmodifier.String{
+						stringplanmodifier.UseStateForUnknown(),
+					},
+				},
+				"invitation_pending": schema.BoolAttribute{
+					MarkdownDescription: "Set to true while the user has not yet accepted their invitation to the organization.",
+					Computed:            true,
+					PlanModifiers: []planmodifier.Bool{
+						boolplanmodifier.UseStateForUnknown(),
+					},
+				},
+				"user_id": schema.StringAttribute{
+					MarkdownDescription: "User ID.",
+					Computed:            true,
+					PlanModifiers: []planmodifier.String{
+						stringplanmodifier.UseStateForUnknown(),
+					},
+				},
+				"organization_role": schema.StringAttribute{
+					MarkdownDescription: "The optional organization role for the member. Can be one of `organization-admin`, `billing-admin`. For more info see: [Organization roles](https://www.elastic.co/guide/en/cloud/current/ec-user-privileges.html#ec_organization_level_roles)",
+					Optional:            true,
+				},
+				"deployment_roles":            deploymentRoleAssignmentsSchema(),
+				"project_elasticsearch_roles": projectElasticsearchRolesSchema(),
+				"project_observability_roles": projectObservabilityRolesSchema(),
+				"project_security_roles":      projectSecurityRolesSchema(),
+			},
+		},
+	}
+}
+
+func deploymentRoleAssignmentsSchema() schema.SetNestedAttribute {
+	return schema.SetNestedAttribute{
+		MarkdownDescription: "Grant access to one or more deployments. For more info see: [Deployment instance roles](https://www.elastic.co/guide/en/cloud/current/ec-user-privileges.html#ec_instance_access_roles).",
+		Optional:            true,
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: map[string]schema.Attribute{
+				"role": schema.StringAttribute{
+					MarkdownDescription: "Assigned role. Must be on of `viewer`, `editor` or `admin`.",
+					Required:            true,
+				},
+				"all_deployments": schema.BoolAttribute{
+					MarkdownDescription: "Role applies to all deployments in the organization.",
+					Optional:            true,
+					PlanModifiers: []planmodifier.Bool{
+						planmodifiers.BoolDefaultValue(false), // consider unknown as false
+					},
+				},
+				"deployment_ids": schema.SetAttribute{
```

Comment:
What happen if the same deployment ID is used in 2 different nested objects? e.g.
first object:
``` 
        {
          role           = "viewer"
          deployment_ids = ["1111"]
        }
```
second object:
``` 
        {
          role           = "admin"
          deployment_ids = ["1111"]
        }
```
Does it make sense to validate cases like this?